pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -297,6 +298,77 @@ func (r *ReconcileStorageCluster) newStorageClasses(initData *ocsv1.StorageClust
 	return ret, nil
 }
 
+// newRBDSnapshotClass returns a new RBD backed SnapshotClass
+func (r *ReconcileStorageCluster) newRBDSnapshotClass(initData *ocsv1.StorageCluster) *snapapi.VolumeSnapshotClass {
+	retSC := &snapapi.VolumeSnapshotClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: generateNameForRBDSnapshotClass(initData),
+		},
+		Driver: generateNameForRBDSnapshotClassDriver(initData),
+		Parameters: map[string]string{
+			"clusterID": initData.Namespace,
+			"csi.storage.k8s.io/snapshotter-secret-name":      "rook-csi-rbd-provisioner",
+			"csi.storage.k8s.io/snapshotter-secret-namespace": initData.Namespace,
+		},
+		DeletionPolicy: snapapi.VolumeSnapshotContentDelete,
+	}
+	return retSC
+}
+
+func (r *ReconcileStorageCluster) newSnapshotClasses(instance *ocsv1.StorageCluster) []*snapapi.VolumeSnapshotClass {
+	scs := []*snapapi.VolumeSnapshotClass{
+		r.newRBDSnapshotClass(instance),
+		// TODO
+		// r.newCephFsSnapshotClass(instance),
+	}
+	return scs
+}
+
+func (r *ReconcileStorageCluster) createSnapshotClasses(scs []*snapapi.VolumeSnapshotClass, reqLogger logr.Logger) error {
```

Comment:
I would prefer `vsc` instead of `scs` as the field name matches.

Reply:
Yup, done...